Mathlib/Computability/TuringDegree.lean

```diff
@@ -7,42 +7,40 @@ import Mathlib.Computability.Partrec
 import Mathlib.Order.Antisymmetrization
 
 /-!
-# Oracle Computability and Turing Degrees
+# Oracle computability and Turing degrees
 
-This file defines a model of oracle computability using partial recursive functions.
-This file introduces Turing reducibility and equivalence, prove that Turing equivalence is an
-equivalence relation, and define Turing degrees as the quotient under this relation.
+This file defines a model of oracle computability using partial recursive functions. It introduces
+Turing reducibility and equivalence, proves that Turing equivalence is an equivalence relation, and
+defines Turing degrees as the quotient under this relation.
 
-## Main Definitions
+## Main definitions
 
-- `RecursiveIn O f`:
-  An inductive definition representing that a partial function `f` is partial recursive given access
-  to a set of oracles O.
-- `TuringReducible`: A relation defining Turing reducibility between partial functions.
-- `TuringEquivalent`: An equivalence relation defining Turing equivalence between partial functions.
-- `TuringDegree`: The type of Turing degrees, defined as the quotient of partial functions under
+* `RecursiveIn O f`: an inductive definition expressing that a partial function `f` is partial
+  recursive given access to a set of oracles `O`.
+* `TuringReducible`: a relation defining Turing reducibility between partial functions.
+* `TuringEquivalent`: an equivalence relation defining Turing equivalence between partial functions.
+* `TuringDegree`: the type of Turing degrees, defined as the quotient of partial functions under
   `TuringEquivalent`.
 
-## Notation
+## Notations
 
-- `f ≤ᵀ g` : `f` is Turing reducible to `g`.
-- `f ≡ᵀ g` : `f` is Turing equivalent to `g`.
+* `f ≤ᵀ g`: `f` is Turing reducible to `g`.
+* `f ≡ᵀ g`: `f` is Turing equivalent to `g`.
 
-## Implementation Notes
+## Implementation notes
 
-The type of partial functions recursive in a set of oracle `O` is the smallest type containing
-the constant zero, the successor, left and right projections, each oracle `g ∈ O`,
-and is closed under pairing, composition, primitive recursion, and μ-recursion.
+The type of partial functions recursive in a set of oracles `O` is the smallest type containing the
+constant zero, the successor, left and right projections, each oracle `g ∈ O`, and is closed under
+pairing, composition, primitive recursion, and μ-recursion.
 
 ## References
 
-* [Odifreddi1989] Odifreddi, Piergiorgio.
-  *Classical Recursion Theory: The Theory of Functions and Sets of Natural Numbers,
-  Vol. I*. Springer-Verlag, 1989.
+* [P. Odifreddi, *Classical Recursion Theory: The Theory of Functions and Sets of Natural Numbers,
```

Comment:
I had to break this reference up over two lines due to length restrictions. I haven't checked whether that breaks anything.